feconf.py

```diff
@@ -470,6 +470,12 @@ def get_empty_ratings():
 # A sorted list of default categories.
 DEFAULT_CATEGORIES = sorted(CATEGORIES_TO_COLORS.keys())
 
+#Size of the gravatar in px assigned to the user after signup
+GRAVATAR_SIZE_PX = 150
+
+#Default Identicon for new users
+DEFAULT_IDENTICON_DATA_URL = '/images/avatar/user_blue_72px.png'
```

Comment:
Yep, I considered computing it on the fly as well, but I agree with you that that overhead is unnecessary. I think a long constant is fine.